be/src/column/column_helper.cpp

```diff
@@ -252,7 +252,7 @@ int64_t ColumnHelper::find_first_not_equal(const Column* column, int64_t target,
 // Nullable(int8), but required return type is nullable(string), so col need align return type to nullable(string).
 ColumnPtr ColumnHelper::align_return_type(const ColumnPtr& old_col, const TypeDescriptor& type_desc, size_t num_rows,
                                           const bool is_nullable) {
-    MutableColumnPtr new_column = old_col->clone();
+    MutableColumnPtr new_column = (std::move(*old_col)).mutate();
```

Comment:
After using std::move(), will there be any problems if the caller of align_return_type uses Column again?

Reply:
`new_column` is  a shadow copy of `old_col`, it's not fine to use it any more. change rvalue reference here.